src/main/java/com/learn/orderit/models/UserRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.learn.orderit.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Past;
+import java.util.Date;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
json request should accept field names in snake_case. You can use Jackson library for the same.